students/olszewski_bartosz/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,19 @@
+a = [int(x) for x in input().split()]
+minimum = a[0]
+maximum = a[0]
+min_index = 0
+max_index = 0
+
+for i in range(len(a)):
+    if a[i] < minimum:
+        minimum = a[i]
+        min_index = i
+    if a[i] > maximum:
+        maximum = a[i]
+        max_index = i
+
+a[max_index] = minimum
```

Comment:
Instead of using additional variable to hold min or max value you can try the following:
a[max_index], a[min_index] = a[min_index], a[max_index]